cmd/main.go

```diff
@@ -80,13 +80,21 @@ func main() {
 	var notificationsTopic = flag.String("notifications", "",
 		"The name of the notification topic in SQS to publish events' outcomes to; if not "+
 			"specified, no outcomes will be published")
+	var acksTopic = flag.String("acks", "",
+		"If `acks` is specified and provided a value, all positive outcomes (Ok) will be published here, and only "+
+			"errors will be published to the `notifications` queue. If specified and not provided a value, "+
+			"Ok outcomes will not be published. If not specified, all results "+
```

Comment:
I don't actually think you can achieve the "specified and not provided a value"?
something like `-ack -foo bar` will assume that the value of `ack` is `-foo` (and the argument `-foo` if required, is missing)

Haven't really tried it, though - have you?
At any rate, I don't like this pattern, as it's confusing.

Please add a boolean `-notify-errors-only` to achieve the same outcome instead (and add a check that it cannot be specified alongside `-ack`)